app/views/contestants/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1>Contestants</h1>
+
+<% @contestants.each do | contestant | %>
+    <section>
+      <h3><%= contestant.name %></h3>
+      <p>Projects: <%= contestant.projects.pluck(:name).join(", ") %> </p>
+    </section>
```

Comment:
We don't want to use AR in our views. We want that to live at the model level. So you would want to create a method in your `contestant` model that does this logic. Then just call that method on your contestant.